pimd/pim_cmd.c

```diff
@@ -8299,11 +8300,12 @@ DEFUN_HIDDEN (interface_ip_igmp_query_max_response_time_dsec,
 
 DEFUN_HIDDEN (interface_no_ip_igmp_query_max_response_time_dsec,
 	      interface_no_ip_igmp_query_max_response_time_dsec_cmd,
-	      "no ip igmp query-max-response-time-dsec",
+	      "no ip igmp query-max-response-time-dsec [(1-65535)]",
 	      NO_STR
 	      IP_STR
 	      IFACE_IGMP_STR
-	      IFACE_IGMP_QUERY_MAX_RESPONSE_TIME_DSEC_STR)
+	      IFACE_IGMP_QUERY_MAX_RESPONSE_TIME_DSEC_STR
+	      IGNORED_IN_NO_STR)
 {
 	char default_query_max_response_time[4];
```

Comment:
Same here: either stringnify define or use `DESTROY` in the YANG op.

Reply:
chnaged to destroy